umh-core/pkg/diagnosis/engine_newengine_test.go

```
@@ -119,14 +124,58 @@ var _ = Describe("NewEngine", func() {
+		_, err := NewEngine(tbl)
+		if err == nil {
+			Fail("expected NewEngine to reject a table-level measurement that sets Against")
+		}
+		Expect(err.Error()).To(ContainSubstring("M-against"))
+		Expect(err.Error()).To(ContainSubstring("only meaningful inside a signal"))
+	})
+
+	It("refuses a table-level measurement that sets Requires, which is only meaningful for an instrument", func() {
+		tbl := validTable([]Signal[snap]{validSignal("A")})
+		tbl.Measurements = []Measurement[snap]{{Name: "M-requires", Extract: extract, Span: 60 * time.Second, Reduction: Last, Requires: []Capability{"psi"}}}
+		_, err := NewEngine(tbl)
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (too many statements above if)

(wsl_v5)